File: 251111/수의 계절은/season-of-num.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim();
const M = Number(input);

if (M >= 3 && M <= 5) {
    console.log('Spring');
} else if (M >= 6 && M <= 8) {
    console.log('Summer');
} else if (M >= 9 && M <= 11) {
    console.log('Fall');
} else {
    console.log('Winter');
}

File: 260312/수의 계절은/season-of-num.py
import sys

input = sys.stdin.readline

M = int(input())
season = None

if 3 <= M <= 5:
    season = 'Spring'
elif 6 <= M <= 8:
    season = 'Summer'
elif 9 <= M <= 11:
    season = 'Fall'
else:
    season = 'Winter'


print(season)